kotlinx-coroutines-core/jvm/test/internal/LockFreeLinkedListAtomicLFStressTest.kt

```diff
@@ -18,6 +18,7 @@ import kotlin.test.*
  */
 class LockFreeLinkedListAtomicLFStressTest {
```

Comment:
Do we still need this test if we require obstruction-freedom in `LockFreeListMultiLincheckTest`?

Reply:
Extra testing will never harm, IMO.

Reply:
The only advantage I see in this obstruction-freedom test (guess, you also test for obstruction-freedom, not for lock-freedom like the test name says) is that it can find bugs caused by weak memory model effects. Should we add this functionality to the stress testing in Lincheck and remove all such tests? I personally find this test quite complicated and would be happy to replace it with a straightforward one someday 🙂 .